lib/components/Announcement/Announcement.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+import { createPortal } from 'react-dom';
+import { useStyles } from 'sku/react-treat';
+
+import { useBoxStyles } from '../Box/useBoxStyles';
+import * as styleRefs from '../HiddenVisually/HiddenVisually.treat';
+
+let announcementCounter = 0;
+
+interface AnnouncementProps {
+  children: string | undefined | null;
```

Comment:
Is it deliberate that this doesn't support multiple child nodes?

Reply:
That's correct. If you ever need multiple announcements in the same component (seems unlikely) you'd render two separate `Announcement` components.

Reply:
But I mean that it'll prevent interpolations like this: `<Announcement>{count} item{count === 1 ? '' : 's'}</Announcement>`

Reply:
Yeah good point. We can open it up.